src/bandwidthRtc.ts

```diff
@@ -21,6 +21,7 @@ class BandwidthRtc {
   constructor(logLevel?: LogLevel) {
     if (logLevel) {
       this.logLevel = logLevel;
+      logger.level = logLevel;
```

Comment:
Might as well call setLogLevel() below and move this code into there

Reply:
Good catch.